recipes/mocsy/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+make
+
+rm -rf $SP_DIR/numpy
```

Comment:
@msarahan I noticed that `numpy` made into the package `tarball` so I am manually removing it here. Is that an artifact of the `makefile` used here or a `conda-build` bug?